lib/ngannotate/processor.rb

```diff
@@ -29,10 +29,14 @@ def force
     # Skip processing in environments where it does not make sense.
     # Override by NG_FORCE=true env variable
     #
-    def skip
+    def skip_by_config
       !force && !::Rails.configuration.ng_annotate.process
     end
 
+    def skip
+      skip_by_config || !@file.start_with?(::Rails.root.to_s)
```

Comment:
> I added also some traces to see what processing is done (see rails logs).

yeah, it looks interesting.
I have may be even more elegant solution...
When you define what files should be precompiled, you can use regexp or lambda like this:
```
    config.assets.precompile << lambda do |filename, path|
      path =~ /frontend/ && !%w(.js .css .htm).include?(File.extname(filename))
    end
```
Let's use lambda instead of plain paths here?